packages/icons/src/git-popover.svg

```diff
@@ -0,0 +1 @@
+<svg width="15" height="15" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M7.121.282A.961.961 0 0 1 7.81 0c.257-.002.505.1.687.281l6.032 6.032a.974.974 0 0 1 0 1.375l-6.032 6.031a.977.977 0 0 1-1.375 0L1.09 7.688a.973.973 0 0 1 0-1.375l4.438-4.438a.515.515 0 0 1 .187-.172L7.12.282ZM13.793 7 7.81 1.016l-1.14 1.14.875.876A.99.99 0 0 1 7.809 3a1 1 0 0 1 .968 1.25l1.781 1.781a1 1 0 1 1-.718.719L8.31 5.219v3.922c.299.174.5.49.5.859a1 1 0 1 1-2 0 .99.99 0 0 1 .5-.86V4.86a.991.991 0 0 1-.5-.86.99.99 0 0 1 .031-.265l-.875-.875L1.825 7l5.984 5.985L13.793 7Z" fill="#25A6F7"/></svg>
```

Comment:
Please follow https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices. 

we shouldn't hardcode width, height and color